pytorch_toolkit/formula_recognition/im2latex/data/utils.py

```diff
@@ -452,10 +452,17 @@ def collate_fn(sign2id, batch, *, batch_transform=None):
 
 def create_list_of_transforms(transforms_list, ovino_ir=False):
     transforms = []
+    bolding_transforms = []
     if transforms_list:
         for transform in transforms_list:
             transform_name = transform.pop('name')
-            transforms.append(TRANSFORMS[transform_name](**transform))
+            transform_prob = transform.pop('prob', 1.0)
+            if transform_name == "TransformRandomBolding":
+                bolding_transforms.append(TRANSFORMS[transform_name](**transform))
+            else:
+                transforms.append(RandomApply([TRANSFORMS[transform_name](**transform)], p=transform_prob))
```

Comment:
Can we use this call alwasy?

Reply:
The logic was hidden into `RandomBoldingTransform` class